nasa-gcn/gcn.nasa.gov
# Circulars: Add In-Page Lucene Documentation

# Description
This adds a brief explanation of Lucene queries to the Circulars Archive page, hidden behind a feature flag. 

![image](https://github.com/user-attachments/assets/bd0187f8-a526-460b-ba3c-0adabb8b273f)

Note: This is a simplified version of #2589 that removes the quick action buttons until the proper implementation can be identified.
# Related Issue(s)
Tracked by https://github.com/nasa-gcn/gcn.nasa.gov/issues/2288

# Testing
1. Navigate to Circulars Archive
2. Click on the "Advanced Search" (Tentative Label) Accordion Menu
3. The menu should expand
